packages/extension/src/components/organisms/Header/Header.tsx

```diff
@@ -56,6 +57,12 @@ export const Header: FC<HeaderProps> = ({ wallet: { name, publicAddress } }) =>
     navigate(LIST_WALLETS_PATH);
   }, [navigate]);
 
+  const abbreviateAddress = (address: string, maxLength = 8) => {
+    if (address.length <= maxLength) return address;
```

Comment:
since this logic is also duplicated in `packages/extension/src/components/pages/ReceivePayment/ReceivePayment.tsx`, we can extract this into a util function (for example in `packages/extension/src/utils/format.ts`), and this code would be unit tested, for example:

```
describe('abbreviateAddress', () => {
  it('should return the full address if its length is less than or equal to the max length', () => {
    const address = 'abcd';
    const result = abbreviateAddress(address);
    expect(result).toBe('abcd');
  });

  it('should abbreviate address correctly if it is longer than the default max length', () => {
    const address = 'abcdefghijklmnop';
    const result = abbreviateAddress(address);
    expect(result).toBe('abcd...mnop');
  });

  it('should abbreviate address correctly if it is longer than the specified max length', () => {
    const address = 'abcdefghijklmno';
    const result = abbreviateAddress(address, 6);
    expect(result).toBe('abc...mno');
  });

  it('should return the full address if its length is equal to the max length', () => {
    const address = 'abcdefgh';
    const result = abbreviateAddress(address);
    expect(result).toBe('abcdefgh');
  });

  it('should handle max lengths that are odd', () => {
    const address = 'abcdefghijklmno';
    const result = abbreviateAddress(address, 7);
    expect(result).toBe('abc...mno');
  });

  it('should return an address with ellipsis in the middle if it is just one character longer than max length', () => {
    const address = 'abcdefghi';
    const result = abbreviateAddress(address);
    expect(result).toBe('abcd...fghi');
  });
});
```